x-pack/plugins/infra/public/alerting/metric_threshold/rule_data_formatters.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ALERT_REASON } from '@kbn/rule-data-utils';
+import { ObservabilityRuleTypeFormatter } from '../../../../observability/public';
+
+export const formatReason: ObservabilityRuleTypeFormatter = ({ fields }) => {
+  const reason = fields[ALERT_REASON] ?? '';
```

Comment:
I think it would make sense to use the `-` introduced in https://github.com/elastic/kibana/issues/106513 here as well
```suggestion
  const reason = fields[ALERT_REASON] ?? '-';
```

Reply:
Yep will do so, thx!